examples/csharp/HelloworldLegacyCsproj/README.md

```diff
@@ -3,6 +3,11 @@ gRPC in 3 minutes (C#)
 
 BACKGROUND
 -------------
+This is a different version of the helloworld example, using the old-style .csproj
+files supported by VS2013 and VS2015 (and older versions of mono).
+You can still use gRPC with the old-style .csproj files, but using the new-style
```

Comment:
nit: I might add to this sentence, e.g. "for instructions on using gRPC with new-style projects, see [../helloworld/READMD.md]" .... just to be clear that this example is for the old-style projects.